lib/simple_form.rb

```diff
@@ -265,6 +265,49 @@ def self.setup
     @@configured = true
     yield self
   end
+
+  # Includes a component to be used by Simple Form. Methods defined in a
+  # component will be exposed to be used in the wrapper as Simple::Components
+  #
+  # Examples
+  #
+  #    # The application needs to tell where the components will be.
+  #    Dir[Rails.root.join('config/initializers/simple_form_components/**/*.rb')].each { |f| require f }
```

Comment:
Would it possible to load components from `app/components/**/*.rb` to have feature parity with custom inputs?

Reply:
It's just a suggestion, the component can be anywhere in your application. You just need to give the correct path to Simple Form.

Reply:
Ok, how do you like the idea suggest loading the components from `app/components` to have feature parity with custom inputs? 🤓

Reply:
Ahh I got it now. We already recommend adding custom inputs inside `app/inputs` 😅. This way, I think it makes sense. 👍